documentation/src/test/java/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java

```diff
@@ -168,23 +168,47 @@ public void id() {
 	}
 
 	@Test
-	public void bool() {
+	public void and() {
 		withinSearchSession( searchSession -> {
-			// tag::bool-or[]
+			// tag::and[]
 			List<Book> hits = searchSession.search( Book.class )
-					.where( f -> f.bool()
-							.should( f.match().field( "title" )
-									.matching( "robot" ) ) // <1>
-							.should( f.match().field( "description" )
-									.matching( "investigation" ) ) // <2>
+					.where( f -> f.and(
+									f.match().field( "title" )
+											.matching( "robot" ), // <1>
+									f.match().field( "description" )
+											.matching( "crime" ) // <2>
+							)
 					)
 					.fetchHits( 20 ); // <3>
-			// end::bool-or[]
+			// end::and[]
+			assertThat( hits )
+					.extracting( Book::getId )
+					.containsExactlyInAnyOrder( BOOK3_ID );
+		} );
+	}
+
+	@Test
+	public void or() {
+		withinSearchSession( searchSession -> {
+			// tag::or[]
+			List<Book> hits = searchSession.search( Book.class )
+					.where( f -> f.or(
+									f.match().field( "title" )
+											.matching( "robot" ), // <1>
+									f.match().field( "description" )
+											.matching( "investigation" ) // <2>
+							)
+					)
+					.fetchHits( 20 ); // <3>
+			// end::or[]
 			assertThat( hits )
 					.extracting( Book::getId )
 					.containsExactlyInAnyOrder( BOOK1_ID, BOOK2_ID, BOOK3_ID );
 		} );
+	}
 
+	@Test
+	public void bool() {
 		withinSearchSession( searchSession -> {
 			// tag::bool-and[]
```

Comment:
Maybe remove that example, as it's no longer present in the documentation?

Reply:
Arf.  I meant to do that!  ;-)